src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const itemsList = [{
+  index: 0,
+  description: 'Finish the to-do list',
+  completed: false,
+},
+{
+  index: 1,
+  description: 'Clean the house',
+  completed: false,
+},
+{
+  index: 2,
+  description: 'wash the dishes',
+  completed: true,
+},
+];
+
+const toDoListContainer = document.querySelector('.to-do-list-container');
+
+const itemsListDisplay = () => {
+  itemsList.forEach((item) => {
+    toDoListContainer.innerHTML += `
+    <li class='to-do-item' value='index'>${item.description}</li>`;
+  });
+};
+
```

Comment:
- [ ] Kindly ensure you style your list to match the wireframe.

  Your design| The wireframe
  :---:|:---:
  ![riow](https://user-images.githubusercontent.com/54848363/214310691-228e4ae7-3027-4294-b281-f9fc0a7076f3.jpg) | ![kdkd](https://user-images.githubusercontent.com/54848363/214310700-27665b82-ffb0-4ba8-a7f3-5e99b0627e9c.jpg)